clients/lsp-netbeans.el

```diff
@@ -0,0 +1,129 @@
+;;; lsp-netbeans.el --- Apache Netbeans Language Server Client settings         -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2021  Tim Felgentreff
+
+;; Author: Tim Felgentreff <timfelgentreff@gmail.com>
+;; Keywords: java groovy graalvm lsp
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+;;; Commentary:
+
+;; LSP and DAP client for the Apache Netbeans LSP server.
+
+;; Currently missing many features that are in the VSCode extension, including, but not limited to:
+;; - All custom commands, such as
+;;     java.build.workspace
+;;     java.generate.getters
+;;     java.generate.setters
+;;     java.generate.getters.setters
+;;     java.generate.implementMethod
+;;     java.generate.overrideMethod
+;;     java.generate.equals.hashCode
+;;     java.generate.delegateMethod
+;;     java.generate.constructor
+;;     java.generate.logger
+;;     java.generate.toString
+;;     graalvm.pause.script
+;;     java.super.implementation
```

Comment:
seems like this file will grow, maybe start lsp-netbeans as a separate package?

Reply:
Agree with that

Reply:
I checked the feature set here - https://marketplace.visualstudio.com/items?itemName=ASF.apache-netbeans-java and I would say that we definitely need this in a separate package. But it is still up to you to decide. @ericdallo or I can create the project under emacs-lsp and make you maintainer if you agree.

Reply:
I agree that probably this should be it's own package. However, I doubt I'll be able to look into this more than a couple of days a month, so I would be a bad maintainer. I guess if no one cares and there's not a lot of issues being opened, it's fine, but otherwise I could be a bad look for the project. Maybe it's then better if I keep it in my private github account, we just put a link there from the readme and this way it won't look bad for the emacs-lsp project when I take days to respond to issues and PRs